pkg/policy/repository.go

```diff
@@ -106,11 +106,43 @@ func (p *policyContext) SetDeny(deny bool) bool {
 	return oldDeny
 }
 
+type PolicyRepository interface {
```

Comment:
This is a huge interface, and many of these are test functions.

Now, I would like to clean many of these up, but that will take some time. However, do you think you can expose only the functions used in `pkg/endpoint` along with `daemon/cmd/policy.go` (etc)? I bet we can get the set of exported functions down to a very small set.

Reply:
I removed all methods that are only used in tests.

There are still a lot of them remaining that are used only in one place, especially in `daemon/cmd`.